[Bug]: bash-lsp "Failed to get cpu info" on deno

### Problem description

[bash-lsp](https://github.com/bash-lsp/bash-language-server) exits when running on deno, but not node. However, the script runs fine on Debian when installing deno via the installation script

### What steps will reproduce the bug?

```sh
pkg i deno
export PATH="$HOME/.deno/bin:$PATH"
deno i -gA npm:bash-language-server
# or any editor with LS client
pkg i helix
# NOTE: I tested a version built by cargo on-device;
# `hx -V` prints "helix 25.07.1 (668bf387)" in my case
hx tmp.sh
# wait a moment,
# then the status-line should print "language server exited"
```

content of `~/.cache/helix/helix.log`:
```
2025-10-22T14:48:51.366 helix_lsp::transport [ERROR] bash-language-server err <- "\u{1b}[0m\u{1b}[1m\u{1b}[31merror\u{1b}[0m: Uncaught (in promise) TypeError: Failed to get cpu info\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mObject.cpus\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:os\u{1b}[0m:\u{1b}[0m\u{1b}[33m67\u{1b}[0m:\u{1b}[0m\u{1b}[33m10\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[36mfile:///data/data/com.termux/files/home/.cache/deno/npm/registry.npmjs.org/fast-glob/3.3.3/out/settings.js\u{1b}[0m:\u{1b}[0m\u{1b}[33m10\u{1b}[0m:\u{1b}[0m\u{1b}[33m31\u{1b}[0m\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mObject.<anonymous>\u{1b}[0m (\u{1b}[0m\u{1b}[36mfile:///data/data/com.termux/files/home/.cache/deno/npm/registry.npmjs.org/fast-glob/3.3.3/out/settings.js\u{1b}[0m:\u{1b}[0m\u{1b}[33m61\u{1b}[0m:\u{1b}[0m\u{1b}[33m3\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mModule._compile\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:module\u{1b}[0m:\u{1b}[0m\u{1b}[33m757\u{1b}[0m:\u{1b}[0m\u{1b}[33m34\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mloadMaybeCjs\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:module\u{1b}[0m:\u{1b}[0m\u{1b}[33m782\u{1b}[0m:\u{1b}[0m\u{1b}[33m10\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mObject.Module._extensions..js\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:module\u{1b}[0m:\u{1b}[0m\u{1b}[33m767\u{1b}[0m:\u{1b}[0m\u{1b}[33m12\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mModule.load\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:module\u{1b}[0m:\u{1b}[0m\u{1b}[33m675\u{1b}[0m:\u{1b}[0m\u{1b}[33m32\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mModule._load\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:module\u{1b}[0m:\u{1b}[0m\u{1b}[33m533\u{1b}[0m:\u{1b}[0m\u{1b}[33m12\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mModule.require\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:module\u{1b}[0m:\u{1b}[0m\u{1b}[33m694\u{1b}[0m:\u{1b}[0m\u{1b}[33m19\u{1b}[0m)\n"
2025-10-22T14:48:51.370 helix_lsp::transport [ERROR] bash-language-server err <- "    at \u{1b}[0m\u{1b}[1m\u{1b}[3mrequire\u{1b}[0m (\u{1b}[0m\u{1b}[36mnode:module\u{1b}[0m:\u{1b}[0m\u{1b}[33m828\u{1b}[0m:\u{1b}[0m\u{1b}[33m16\u{1b}[0m)\n"
2025-10-22T14:48:51.442 helix_lsp [ERROR] failed to initialize language server: server closed the stream
2025-10-22T14:48:51.442 helix_lsp::transport [ERROR] bash-language-server err: <- StreamClosed

```

### What is the expected behavior?

It should run like on Debian, or when using Node on Termux.

It's worth noting that `tsserver` and `typescript-language-server` run fine, both on Termux and Debian, node and deno alike

### System information

```shell
Termux Variables:
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP__APK_FILE=/data/app/~~PpgH0Nd9KnXWBibswESkKw==/com.termux-pnnreuFsQ-0LQ2GKo0Ksgw==/base.apk
TERMUX_APP__APK_RELEASE=GITHUB
TERMUX_APP__APP_VERSION_CODE=1022
TERMUX_APP__APP_VERSION_NAME=0.119.0-beta.3
TERMUX_APP__DATA_DIR=/data/user/0/com.termux
TERMUX_APP__IS_DEBUGGABLE_BUILD=true
TERMUX_APP__IS_INSTALLED_ON_EXTERNAL_STORAGE=false
TERMUX_APP__LEGACY_DATA_DIR=/data/data/com.termux
TERMUX_APP__PACKAGE_NAME=com.termux
TERMUX_APP__PID=12082
TERMUX_APP__SE_FILE_CONTEXT=u:object_r:app_data_file:s0:c225,c256,c512,c768
TERMUX_APP__SE_INFO=default:targetSdkVersion=28:complete
TERMUX_APP__TARGET_SDK=28
TERMUX_VERSION=0.119.0-beta.3
TERMUX__APPS_DIR=/data/user/0/com.termux/termux/apps
TERMUX__HOME=/data/data/com.termux/files/home
TERMUX__PREFIX=/data/data/com.termux/files/usr
TERMUX__ROOTFS_DIR=/data/data/com.termux/files
TERMUX__SE_PROCESS_CONTEXT=u:r:untrusted_app_27:s0:c225,c256,c512,c768
TERMUX__UID=10225
TERMUX__USER_ID=0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://mirrors.in.sahilister.net/termux/termux-main/ stable main
Updatable packages:
glib/stable 2.86.1 aarch64 [upgradable from: 2.86.0]
libunbound/stable 1.24.1 aarch64 [upgradable from: 1.24.0]
termux-tools version:
1.45.0
Android version:
14
Kernel build information:
Linux localhost 5.10.198-android12-9-00085-g226a9632f13d-ab11136126 #1 SMP PREEMPT Wed Nov 22 14:16:37 UTC 2023 aarch64 Android
Device manufacturer:
TCL
Device model:
T509A
Supported ABIs:
SUPPORTED_ABIS: arm64-v8a,armeabi-v7a,armeabi
SUPPORTED_32_BIT_ABIS: armeabi-v7a,armeabi
SUPPORTED_64_BIT_ABIS: arm64-v8a
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec-ld-preload.so
Installed termux plugins:
com.termux.api versionCode:1002
```

--- a/packages/deno/0008-os.cpus-permission.patch
+++ b/packages/deno/0008-os.cpus-permission.patch
@@ -1,0 +1,21 @@
+https://github.com/termux/termux-packages/issues/26970
+
+--- a/ext/node/ops/os/mod.rs
++++ b/ext/node/ops/os/mod.rs
+@@ -276,7 +276,15 @@
+     permissions.check_sys("cpus", "node:os.cpus()")?;
+   }
+ 
+-  cpus::cpu_info().ok_or(OsError::FailedToGetCpuInfo)
++  #[cfg(target_os = "android")]
++  {
++    return Ok(cpus::cpu_info().unwrap_or_default());
++  }
++
++  #[cfg(not(target_os = "android"))]
++  {
++    return cpus::cpu_info().ok_or(OsError::FailedToGetCpuInfo);
++  }
+ }
+ 
+ #[op2(stack_trace)]

--- a/packages/deno/build.sh
+++ b/packages/deno/build.sh
@@ -3,6 +3,7 @@ TERMUX_PKG_DESCRIPTION="A modern runtime for JavaScript and TypeScript"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@licy183"
 TERMUX_PKG_VERSION=1:2.5.4
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=(
 	https://github.com/denoland/deno/releases/download/v${TERMUX_PKG_VERSION:2}/deno_src.tar.gz
 	https://github.com/termux/deno-snapshot/releases/download/v${TERMUX_PKG_VERSION:2}/deno-snapshot-aarch64-linux-android-${TERMUX_PKG_VERSION:2}.tar.bz2
@@ -138,7 +139,7 @@ use_jumbo_build=true
 
 	export V8_FROM_SOURCE=1
 	# TODO: How to track the output of v8's build.rs without passing `-vv`
-	cargo build --jobs "${TERMUX_PKG_MAKE_PROCESSES}" --target "${CARGO_TARGET_NAME}" --release -vv
+	cargo build --jobs "${TERMUX_PKG_MAKE_PROCESSES}" --target "${CARGO_TARGET_NAME}" --release
 
 	unset BINDGEN_EXTRA_CLANG_ARGS "$env_name" V8_FROM_SOURCE
 	touch "$__SRC_DIR"/.built

